File: app/src/main/java/com/example/pueblos/InicioActivity.java
package com.example.pueblos;

import android.content.Intent;
import android.graphics.Color;
import android.os.Bundle;
import android.widget.ImageView;
import androidx.appcompat.app.AppCompatActivity;
import androidx.core.content.ContextCompat;

public class InicioActivity extends AppCompatActivity {

    private ImageView btnPueblo;
    private ImageView btnAtractivos;
    private ImageView btnEventos;
    private ImageView btnHotel;
    private ImageView btnRestaurantes;
    private ImageView btnRutas;
    private ImageView btnFeria;
    private ImageView btnPaste;
    private ImageView btnPlata;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_inicio);

        // Inicializa los botones circulares
        btnPueblo = findViewById(R.id.btnPueblo);
        btnAtractivos = findViewById(R.id.btnAtractivos);
        btnEventos = findViewById(R.id.btnEventos);
        btnHotel = findViewById(R.id.btnHotel);
        btnRestaurantes = findViewById(R.id.btnRestaurantes);
        btnRutas = findViewById(R.id.btnRutas);

        // Inicializa las imágenes de eventos
        btnFeria = findViewById(R.id.btnFeria);
        btnPaste = findViewById(R.id.btnPaste);
        btnPlata = findViewById(R.id.btnPlata);

        // Asigna listeners a cada botón circular
        btnEventos.setOnClickListener(v -> {
            abrirActividad(InicioActivity.class);
            btnEventos.setColorFilter(ContextCompat.getColor(this, R.color.rosa));
        });
        btnPueblo.setOnClickListener(v -> {
            abrirActividad(MainActivity.class);
            btnPueblo.setColorFilter(ContextCompat.getColor(this, R.color.rosa));
        });
        btnAtractivos.setOnClickListener(v -> {
            abrirActividad(AtractivosActivity.class);
            btnAtractivos.setColorFilter(ContextCompat.getColor(this, R.color.rosa));
        });


        // Asigna listeners a las imágenes de eventos
        btnFeria.setOnClickListener(v -> abrirActividad(FeriaActivity.class));
        btnPaste.setOnClickListener(v -> abrirActividad(PasteActivity.class));
        btnPlata.setOnClickListener(v -> abrirActividad(InfoActivity.class));
    }

    private void abrirActividad(Class<?> actividad) {
        Intent intent = new Intent(InicioActivity.this, actividad);
        startActivity(intent);
    }
}

File: app/src/main/java/com/example/pueblos/InicioActivity.kt
package com.example.pueblos

import android.content.Intent
import android.os.Bundle
import android.widget.ImageView
import androidx.appcompat.app.AppCompatActivity

class InicioActivity : AppCompatActivity() {

    private lateinit var btnPueblo: ImageView
    private lateinit var btnAtractivos: ImageView
    private lateinit var btnEventos: ImageView
    private lateinit var btnHotel: ImageView
    private lateinit var btnRestaurantes: ImageView
    private lateinit var btnRutas: ImageView
    private lateinit var btnFeria: ImageView
    private lateinit var btnPaste: ImageView
    private lateinit var btnPlata: ImageView

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_inicio)

        // Inicializa los botones circulares
        btnPueblo = findViewById(R.id.btnPueblo)
        btnAtractivos = findViewById(R.id.btnAtractivos)
        btnEventos = findViewById(R.id.btnEventos)
        btnHotel = findViewById(R.id.btnHotel)
        btnRestaurantes = findViewById(R.id.btnRestaurantes)
        btnRutas = findViewById(R.id.btnRutas)

        // Inicializa las imágenes de eventos
        btnFeria = findViewById(R.id.btnFeria)
        btnPaste = findViewById(R.id.btnPaste)
        btnPlata = findViewById(R.id.btnPlata)

        // Asigna listeners a cada botón circular
        btnPueblo.setOnClickListener {
            abrirActividad(PueblosActivity::class.java)
        }
        btnAtractivos.setOnClickListener {
            abrirActividad(AtractivosActivity::class.java)
        }
        btnEventos.setOnClickListener {
            abrirActividad(EventosActivity::class.java)
        }
        btnHotel.setOnClickListener {
            abrirActividad(HotelActivity::class.java)
        }
        btnRestaurantes.setOnClickListener {
            abrirActividad(RestaurantesActivity::class.java)
        }
        btnRutas.setOnClickListener {
            abrirActividad(RutasActivity::class.java)
        }

        // Asigna listeners a las imágenes de eventos
        btnFeria.setOnClickListener {
            abrirActividad(FeriaActivity::class.java)
        }
        btnPaste.setOnClickListener {
            abrirActividad(PasteActivity::class.java)
        }
        btnPlata.setOnClickListener {
            abrirActividad(PlataActivity::class.java)
        }
    }

    class PlataActivity {

    }

    class PasteActivity {

    }

    class FeriaActivity {

    }

    class RutasActivity {

    }

    class RestaurantesActivity {

    }

    class HotelActivity {

    }

    class EventosActivity {

    }

    class AtractivosActivity {

    }

    class PueblosActivity {

    }

    private fun abrirActividad(actividad: Class<*>) {
        val intent = Intent(this, actividad)
        startActivity(intent)
    }
}
